server/src/ScriptRunner/Scripts/index/add_not_investigated_to__investigation_statuses.sql

```diff
@@ -0,0 +1,27 @@
+INSERT INTO investigation_status (display_name, id)
+SELECT 'לא נחקר',100000005
+WHERE
```

Comment:
the id is not unique and then lunes 3-6 are unnecessary?

Reply:
@ornahasson read about ON CONFLICT:
https://www.postgresql.org/docs/9.5/sql-insert.html